<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>300</height>
   </rect>
  </property>
  <property name="mouseTracking">
   <bool>false</bool>
  </property>
  <property name="tabletTracking">
   <bool>false</bool>
  </property>
  <property name="focusPolicy">
   <enum>Qt::NoFocus</enum>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::NoContextMenu</enum>
  </property>
  <property name="acceptDrops">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="toolTip">
   <string/>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <property name="inputMethodHints">
   <set>Qt::ImhNone</set>
  </property>
  <property name="toolButtonStyle">
   <enum>Qt::ToolButtonTextOnly</enum>
  </property>
  <property name="animated">
   <bool>true</bool>
  </property>
  <property name="documentMode">
   <bool>false</bool>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Rounded</enum>
  </property>
  <property name="dockNestingEnabled">
   <bool>false</bool>
  </property>
  <property name="dockOptions">
   <set>QMainWindow::AllowTabbedDocks|QMainWindow::AnimatedDocks</set>
  </property>
  <property name="unifiedTitleAndToolBarOnMac">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QCheckBox" name="checkBoxPanel">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>70</y>
      <width>151</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Courier New</family>
      <pointsize>11</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="toolTip">
     <string>在屏幕右上角显示打卡查询结果，同时显示下班时间</string>
    </property>
    <property name="statusTip">
     <string/>
    </property>
    <property name="text">
     <string> 显示查询结果</string>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonTray">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>230</y>
      <width>90</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Courier New</family>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="toolTip">
     <string>隐藏窗口到任务栏图标</string>
    </property>
    <property name="text">
     <string>隐藏窗口</string>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <property name="autoExclusive">
     <bool>false</bool>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QComboBox" name="comBoxRemindTime">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>160</y>
      <width>141</width>
      <height>28</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="tabletTracking">
     <bool>false</bool>
    </property>
    <property name="focusPolicy">
     <enum>Qt::WheelFocus</enum>
    </property>
    <property name="acceptDrops">
     <bool>false</bool>
    </property>
    <property name="toolTip">
     <string>选择提醒时间之小时</string>
    </property>
    <property name="statusTip">
     <string/>
    </property>
    <property name="inputMethodHints">
     <set>Qt::ImhNone</set>
    </property>
    <property name="editable">
     <bool>false</bool>
    </property>
    <property name="currentIndex">
     <number>6</number>
    </property>
    <property name="maxVisibleItems">
     <number>10</number>
    </property>
    <property name="insertPolicy">
     <enum>QComboBox::InsertAtBottom</enum>
    </property>
    <property name="sizeAdjustPolicy">
     <enum>QComboBox::AdjustToContentsOnFirstShow</enum>
    </property>
    <property name="duplicatesEnabled">
     <bool>false</bool>
    </property>
    <property name="frame">
     <bool>true</bool>
    </property>
    <item>
     <property name="text">
      <string>     08:00:00</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>     08:05:00</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>     08:10:00</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>     08:15:00</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>     08:20:00</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>     08:25:00</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>     08:30:00</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>     08:35:00</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>     08:40:00</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>     08:45:00</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>     08:50:00</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>     08:55:00</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>     09:00:00</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>     09:05:00</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>     09:10:00</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="labelTitle">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>581</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Courier New</family>
      <pointsize>16</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>打  卡  查  询  助  手</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignBottom|Qt::AlignHCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditName">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>70</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>11</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="toolTip">
     <string/>
    </property>
    <property name="statusTip">
     <string>上网认证账号</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="frame">
     <bool>false</bool>
    </property>
    <property name="dragEnabled">
     <bool>true</bool>
    </property>
    <property name="placeholderText">
     <string>上网认证账号</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditPwd">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>110</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="toolTip">
     <string/>
    </property>
    <property name="statusTip">
     <string>上网认证密码</string>
    </property>
    <property name="whatsThis">
     <string/>
    </property>
    <property name="inputMethodHints">
     <set>Qt::ImhNoAutoUppercase|Qt::ImhNoPredictiveText|Qt::ImhSensitiveData</set>
    </property>
    <property name="inputMask">
     <string/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="frame">
     <bool>false</bool>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::PasswordEchoOnEdit</enum>
    </property>
    <property name="cursorPosition">
     <number>0</number>
    </property>
    <property name="dragEnabled">
     <bool>false</bool>
    </property>
    <property name="placeholderText">
     <string>上网认证密码</string>
    </property>
    <property name="cursorMoveStyle">
     <enum>Qt::LogicalMoveStyle</enum>
    </property>
    <property name="clearButtonEnabled">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonQuit">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>230</y>
      <width>90</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Courier New</family>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="toolTip">
     <string/>
    </property>
    <property name="text">
     <string>退出程序</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBoxWebAuth">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>160</y>
      <width>151</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Courier New</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="toolTip">
     <string>未设置用户名或密码，不能选择此项</string>
    </property>
    <property name="text">
     <string> 自动上网认证</string>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBoxRemindOnlyWorkDay">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>100</y>
      <width>151</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Courier New</family>
      <pointsize>11</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="toolTip">
     <string>周一到周五提醒，周六、周日不提醒</string>
    </property>
    <property name="text">
     <string> 仅工作日提醒</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBoxAutoRun">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>130</y>
      <width>151</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Courier New</family>
      <pointsize>11</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="toolTip">
     <string/>
    </property>
    <property name="text">
     <string> 开机自动运行</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonSaveConfig">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>230</y>
      <width>90</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Courier New</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="toolTip">
     <string>保存配置文件da.xml到当前运行目录下</string>
    </property>
    <property name="text">
     <string>保存配置</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonAbout">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>230</y>
      <width>90</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Courier New</family>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="toolTip">
     <string/>
    </property>
    <property name="text">
     <string>关 于</string>
    </property>
   </widget>
   <zorder>checkBoxPanel</zorder>
   <zorder>comBoxRemindTime</zorder>
   <zorder>labelTitle</zorder>
   <zorder>lineEditName</zorder>
   <zorder>lineEditPwd</zorder>
   <zorder>pushButtonTray</zorder>
   <zorder>checkBoxWebAuth</zorder>
   <zorder>checkBoxRemindOnlyWorkDay</zorder>
   <zorder>pushButtonQuit</zorder>
   <zorder>checkBoxAutoRun</zorder>
   <zorder>pushButtonSaveConfig</zorder>
   <zorder>pushButtonAbout</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusBar">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="sizeGripEnabled">
    <bool>false</bool>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
